plugins/woocommerce/includes/class-wc-emails.php

```diff
@@ -24,6 +24,14 @@ class WC_Emails {
 	 */
 	public $emails = array();
 
+
+	/**
+	 * Array of email notification classes and paths
+	 *
+	 * @var WC_Email[]
+	 */
+	public $default_emails_paths = array();
```

Comment:
Given we would now have a public method that returns the list of default e-mail classnames and their paths, I don't think we need to also expose it as a public var in the class.
In the end, what matters is what ends up in the `$emails` array, and not necessarily the intermediate steps.

Do let me know if you think I'm missing something and you feel this var could be useful to have.